src/Websites/Websites.Autorest/docs/Get-AzStaticWebApp.md

```diff
@@ -190,13 +190,16 @@ INPUTOBJECT <IWebsitesIdentity>: Identity Parameter
   - `[DomainName <String>]`: The custom domain name.
   - `[EnvironmentName <String>]`: The stage site identifier.
   - `[FunctionAppName <String>]`: Name of the function app registered with the static site build.
-  - `[Id <String>]`: Resource identity path
+  - `[Id <String>]`: History ID.
```

Comment:
Looks like `Id` is changed to `Id1`. This may cause issue.

Reply:
Done